x/ccv/provider/types/msg.go

```diff
@@ -76,8 +75,9 @@ func (msg MsgAssignConsumerKey) ValidateBasic() error {
 	if 128 < len(msg.ChainId) {
 		return ErrBlankConsumerChainID
 	}
-	if msg.ProviderAddr == "" {
```

Comment:
Do not remove this check?

Reply:
https://github.com/cosmos/cosmos-sdk/blob/f771f20da49afc0fa3e0e75da431733382ddf3dd/types/address.go#L325-L328